From 042b4ea959cde508cf1ee7fe776dba58994f3657 Mon Sep 17 00:00:00 2001
From: Antonio Borneo <antonio.borneo@foss.st.com>
Date: Wed, 5 Feb 2025 12:15:13 +0100
Subject: [PATCH 1639/1695] dts: stm32: use stm32-irq-notifier instead of
 stm32mp1-pwr-irq-user

Use the new driver.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git f8e66a3c91ec1b008d06f280b8e3229622ab8f32 dts: stm32: use stm32-irq-notifier instead of stm32mp1-pwr-irq-user]
Signed-off-by: Antonio Borneo <antonio.borneo@foss.st.com>
Change-Id: I079b405ec50c96e54a10f0dbb92ecb339ee579e1
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/440870
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 core/arch/arm/dts/stm32mp135f-dk.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/core/arch/arm/dts/stm32mp135f-dk.dts b/core/arch/arm/dts/stm32mp135f-dk.dts
index bf97d9217..082d9fd78 100644
--- a/core/arch/arm/dts/stm32mp135f-dk.dts
+++ b/core/arch/arm/dts/stm32mp135f-dk.dts
@@ -34,7 +34,7 @@
 	};
 
 	magic_wol: magic_wol {
-		compatible = "st,stm32mp1-pwr-irq-user";
+		compatible = "st,stm32-irq-notifier";
 		interrupts-extended = <&exti 56 IRQ_TYPE_EDGE_FALLING>;
 		wakeup-source;
 		status = "okay";
@@ -67,7 +67,7 @@
 	};
 
 	typec_wakeup: typec_wakeup {
-		compatible = "st,stm32mp1-pwr-irq-user";
+		compatible = "st,stm32-irq-notifier";
 		interrupts-extended = <&exti 59 IRQ_TYPE_EDGE_FALLING>;
 		wakeup-source;
 		st,notif-it-id = <1>;
-- 
2.47.2

